x-pack/platform/plugins/shared/cases/server/services/cases/sanitizers.ts

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
I created this sanitizers file for functions that clean request params. I saw there is also transformations but I think it's not the same. I also though it's important to add versioning to this functions as if there is a breaking change in the param, there would also be one here. Let me know if this should be done somewhere else.

Reply:
I'm thinking about moving this to `.../cases/server/client/cases/` instead. I won't have to update anything, in `.../cases/server/client/` it should be something more generic as there are assignees, at least, in user-actions. Seems like we follow the same pattern for validators too.

Reply:
Sounds good to me!